notebooks/nessie-delta-demo-nba.ipynb

```diff
@@ -11,7 +11,7 @@
     "Initialize Pyspark\n",
     "----------------------------------------------\n",
     "To get started, we will first have to do a few setup steps that give us everything we need\n",
-    "to get started with Nessie. In case you're interested in the detailed setup steps for Spark, you can check out the [docs](https://projectnessie.org/tools/spark/)\n",
+    "to get started with Nessie. In case you're interested in the detailed setup steps for Spark, you can check out the [docs](https://projectnessie.org/tools/iceberg/spark/)\n",
```

Comment:
This should be delta's link, not iceberg

https://projectnessie.org/tools/deltalake/spark/